doc/getting_started/installation_win.rst

```diff
@@ -20,6 +20,144 @@ Windows system with the latest updates installed.
 Installing Requirements and Dependencies
 ****************************************
 
+There are 3 different ways of developing for Zephyr on Microsoft Windows.
+The first one is fully Windows native, whereas the 2 additional ones require
+emulation layers that slow down build times and are not as optimal. All of
+them are presented here for completeness, but unless you have a particular
+requirement for a UNIX tool that is not available on Windows, we strongly
+recommend you use the Windows Command Prompt for performance and minimal
+dependency set.
+
+Using Windows Command Prompt (Recommended, fastest)
+===================================================
+
+The easiest way to install the dependencies natively on Microsoft Windows is
+to use the :program:`Chocolatey` package manager (`Chocolatey website`_).
+If you prefer to install those manually then simply download the required
+packages from their respective websites.
+
+   .. note::
+        There are multiple ``set`` statements in this tutorial. You can avoid
+        typing them every time by placing them inside a ``.cmd`` file and
+        running that every time you open a Command Prompt.
+
+#. If you're behind a corporate firewall, you'll likely need to specify a
+   proxy to get access to internet resources::
+
+      > set HTTP_PROXY=http://user:password@proxy.mycompany.com:1234
```

Comment:
also need to set HTTPS_PROXY (used in the git command below)

Reply:
OK! will fix that